next.config.js

```diff
@@ -55,6 +55,10 @@ const permanentRedirectArray = [
     "/price-feeds/pythnet-price-feeds/best-practices",
     "/price-feeds/best-practices",
   ],
+  [
+    "/price-feeds/pythnet-price-feeds/pull-updates",
+    "/price-feeds/pull-updates",
+  ],
```

Comment:
drive-by: we moved this page and forgot a redirect. There are a couple inbound links to the old URL.